### ISSUE TITLE
NullPointerException during creating paymetn

### ISSUE THREAD

**Issue by bormanEcwid** [paypal/PayPal-REST-API-issues#213]
### General information

* Environment: Sandbox and Production
* `PayPal-Debug-ID` values: no PayPal-Debug-ID, endpoint return only exception text
* Language, language version, and OS: 

### Issue description

Paypal returns 500 error when I try to create payment by REST API (https://developer.paypal.com/docs/api/payments/#payment_create)

REQUEST body:
`
{
"application_context": {
"brand_name": "My store name",
"locale": "en_GB",
"shipping_preference": "NO_SHIPPING",
"user_action": "commit"
},
"intent": "sale",
"payer": {
"payment_method": "paypal",
"payer_info": {
"billing_address": {
"line1": "Hoendenberg kaya N 46",
"city": "Curacao",
"country_code": "CW",
"postal_code": "0000 CW",
"state": "Netherlands Antilles"
}
}
},
"transactions": [
{
"reference_id": "1584002-1421002",
"amount": {
"currency": "USD",
"total": "540.00",
"details": {
"subtotal": "555.00",
"shipping": "0.00",
"tax": "0.00",
"shipping_discount": "15.00"
}
},
"payee": {
"email": "my_test@email.com"
},
"description": "Order from test_eml@tst.ru",
"note_to_payee": "tst",
"custom": "1584002",
"item_list": {
"items": [
{
"sku": "3w23",
"name": "test",
"quantity": "1",
"price": "555.00",
"currency": "USD",
"tax": "0.00"
}
]
},
"notify_url": "http://tst.com",
"order_url": "http://tst.com"
}
],
"redirect_urls": {
"return_url": "http://tst.com",
"cancel_url": "http://tst.com"
}
}`

RESPONSE:
status: 500
body:` javax.validation.ValidationException: HV000028: Unexpected exception during isValid call.[java.lang.NullPointerException]`

 No difference how I do request:
- by your skd (https://github.com/paypal/PayPal-Java-SDK)
- by postman
- by curl
Always I get 500 error whithout PayPal-Debug-ID, whithout json-response. PayPal returns only text whith exception.

When I replace value of field "payer -> payer_info -> country_code" from "CW" (Curaçao) to other ( for example "CU" Cuba - it works correct, PayPal returns 201 status). 

I think data shouldn't raise internal error. Data can raise 400 error not 500. Also for any error PayPal should return json in response body.

**Comment by xiaoleih41** [comment]
I agree that this should be a data error with http 400 error. Please check [here](https://developer.paypal.com/docs/classic/api/country_codes/) for PayPal supported country list

**Comment by juvirez** [comment]
The problem continues to be reproduced.
I get an internal error with body "Unexpected 'j'", when I sending an unsupported country code.
